### DISCUSSION TITLE
All the GSSoC'21 Contributors

### DISCUSSION THREAD

**Discussion by vatsalkesarwani12** [Algo-Phantoms/Algo-Tree#19] [Q&A] [answered]
When to start contributing?

**Comment by vatsalkesarwani12** [comment, marked as answer]
First of all, Welcome to Algo Phantoms👋

All the contribution to Algo-Tree are welcome but contributing to making a leaderboard in GSSoC'21 you must raise PR's from `1st March` to `31st May`. 

Thank you :)
Happy Coding !!

**Comment by slashharsh** [comment]
What are the supported languages?

**Reply by vatsalkesarwani12** [reply]
Java, c++, python

**Reply by slashharsh** [reply]
Thanks!

**Comment by Op-panda** [comment]
should I write proposal before start contributing to Orgs.??

**Reply by vatsalkesarwani12** [reply]
Not required man, just read the readme and get started 👍🏻

**Comment by ShubhamGupta577** [comment]
Should I have to wait for the assignment of an issue or I can start with direct PR?

**Reply by vatsalkesarwani12** [reply]
You have to wait for assignment of issue. Since official contribution date is 8th be a little patient 👍

**Comment by aman-simplycool** [comment]
also want to fix an issue but i am not getting that whether u will provide us an issue or we will be searching for any issue

**Reply by MukulKolpe** [reply]
Yes, I also have the same doubt. I was wondering, should we try to raise an issue and ask them to assign us, or did they have planned to upgrade the codebase so that more people can work on this project?

**Reply by vatsalkesarwani12** [reply]
You can do either way

**Comment by rutuja2229** [comment]
> When to start contributing?

I want to ask here that how will u get to that we are GSSoC participants and assigns the issue, bcz others can also grab this opportunity

**Reply by vatsalkesarwani12** [reply]
That our problem, you guys just focus on this best opportunity to learn👍

**Reply by MukulKolpe** [reply]
Ok, Thank You!!!

**Comment by nagasaisriya** [comment]
Hello, Can Java Applets(GUI) codes be added to this project?
Would like to know if I can add Java GUI programs that involves awt and swing components or only programs related to Algorithms/Data Structures are accepted.
Thank you!

**Reply by vatsalkesarwani12** [reply]
No sorry, not accepting that.
